superset-frontend/plugins/plugin-chart-echarts/src/Treemap/transformProps.ts

```diff
@@ -259,7 +263,7 @@ export default function transformProps(
         show: false,
       },
       itemStyle: {
-        color: CategoricalColorNamespace.getColor(),
+        color: '#1FA8C9',
```

Comment:
Hey @stephenLYZ can you shed some light on this change for me, please?

Reply:
The logic here is to display a default color when the treemap has no value, otherwise it is blank and the user may thinks it crashes. The previous way would generate a random color and give the scale an `undefined` key. That's way were wired.I think it is enough to write as one fixed color here.